.idea/runConfigurations/_template__of_Gradle.xml

```diff
@@ -16,7 +16,6 @@
     <ExternalSystemDebugServerProcess>true</ExternalSystemDebugServerProcess>
     <ExternalSystemReattachDebugProcess>true</ExternalSystemReattachDebugProcess>
     <DebugAllEnabled>false</DebugAllEnabled>
-    <RunAsTest>false</RunAsTest>
```

Comment:
I guess, this change is not intended.

Reply:
If the change is not intended, then please roll it back :)